dsn-2018/bare_conf.tex

```diff
@@ -0,0 +1,1350 @@
+    \item We propose five universal metrics for evaluating the utility of
+        a class of collisions.
+\end{itemize}
+We propose that collision classes of high utility are closer to a second
+preimage attack in the amount of flexibility they provide to an attacker,
+whereas classes of low utility may not affect all systems which use a
+particular hash function.
+
+\textbf{TODO - Finish Organization}
+The remainder of this paper is organized as fol
+lows. In Section
+\ref{Sec:Related} we discuss previous results and show how our research
+expands upon prior work. In Section \ref{Sec:Terminology}, we discuss the
+terminology and notations we use throughout the remainder of the paper.
+In Section \ref{Sec:Intuition}, we give the intuition between measuring
+the utility of a collision and motivate why it is useful. In Section
+\ref{Sec:Logical}, we present new techniques in logical cryptanalysis to
+analyze collisions in hash functions.
+
+
+
+
+
+
+\section{Related Work} \label{Sec:Related}
+
+This paper draws heavily upon the public cryptanalysis of MD4, the wide
+availability of high quality SAT solvers, and previous work in logical
+cryptanalysis. In particular, we look at ways that SAT solving can aid
+cryptanalysis instead of using cryptanalysis to benchmark SAT solvers. We
+then apply them to aid the understanding of attacks against real world
+systems, and propose general metrics that target different types of systems.
+Towards this, we look at prior work in two main areas: differential and logical
+cryptanalysis.
+
+\subsection{On Differential Cryptanalysis}
+
+Differential Cryptanalysis has been the major technique behind the discovery
+of collisions in cryptographic hash functions. Its importance can be seen
+everywhere from X. Wang's attacks on MD4 \cite{Wang2005} to more recent
+cryptanalysis of general purpose hash functions such as Murmur3 by
+J. Aumasson, D. Bernstein, and M. Bo{\ss}let \cite{murmur3DC}. However, while
+differential cryptanalysis plays an important role, techniques for automated
+analysis of hash functions are still an emerging area. Further, techniques
+are being developed to make hash functions impervious to differential
+analysis.
+
+Our work removes the need for finding message modification techniques by
+specifying the differential path as part of the SAT formula and letting the
+solver find a pair of satisfying messages which follow the differential path
+and produce a collision. Further constraints can then be placed upon this
+model, such as a chosen prefix or desired start state. We believe performing
+these attacks by hand to be difficult, and note that developed message
+modification techniques may affect chosen prefixes. Thus, we seek to develop
+techniques to replace differential cryptanalysis with logical cryptanalysis.
+
+
+\subsection{On Logical Cryptanalysis}
+
+Logical Cryptanalysis and its encoding as SAT likely started under the work of
```

Comment:
Here you capitalize logical cryptanalysis, but not above. Please choose one formatting and stick to it.

Reply:
whichever you choose, make sure to fix it throughout the whole paper too